resources/type-system/inference.md

```diff
@@ -1026,15 +1026,15 @@ inference depends on the context `K`, as well as the flow analysis state._
 
 A property of expression inference, known as _soundness_, is that when an
 elaborated expression is executed, it is guaranteed either to diverge, throw an
-exception, or evaluate to a value that is an _instance satisfying_ its static
-type. _Instance satisfying_ is defined as follows: a value `v` is an instance
-satisfying type `T` iff the runtime type of `v` is a subtype of the _extension
+exception, or evaluate to a value that is an _instance implementing_ its static
+type. _Instance implementing_ is defined as follows: a value `v` is an instance
+implementing type `T` iff the dynamic type of `v` is a subtype of the _extension
 type erasure_ of `T`. _So, for example, every value is considered an instance
-satisfying type `dynamic`, and all values except `null` are considered an
-instance satisfying type `Object`._
+implementing type `dynamic`, and all values except `null` are considered an
```

Comment:
(Since `dynamic` is not an interface, you can't have it as super-interface, so you can't implement it. But everything is a subtype of `dynamic` because it's a top type.)